x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/inference/trainedmodel/LearningToRankConfigTests.java

```diff
@@ -140,6 +143,16 @@ public void testDuplicateFeatureNames() {
         expectThrows(IllegalArgumentException.class, () -> builder.build());
     }
 
+    public void testLicenseSupport_ForPutAction_RequiresEnterprise() {
+        var config = randomLearningToRankConfig();
+        assertThat(config.getMinLicenseSupportedForAction(RestRequest.Method.PUT), is(License.OperationMode.ENTERPRISE));
+    }
+
+    public void testLicenseSupport_ForGetAction_RequiresPlatinum() {
```

Comment:
Is this platinum tester needed?

Reply:
I don't think it's 100% necessary as tests are also covered in the YAML specs... however, I felt that as long as I had an explicit test for the PUT with Enterprise - I should have an additional localized check for at least one other method... 🤷